set.mm

```diff
@@ -421310,6 +421439,17 @@ corresponds to the size of the set of closed walks of length n (in an
       isclwlk ancom bitri hash0 eqcomi fveq2i biantrur 3bitr4g ) BDFZGAHIJHZAHK
       ZIABLHMZNZUKGGOPCAQZNIABRHMZUNUIULUNUKADBCESTUOULUKNUMAIBUAULUKUBUCUKUNGU
       JAUJGUDUEUFUGUH $.
+
+    $( Any vertex (more precisely, a pair of an empty set (of edges) and a
+       singleton function to this vertex) determines a closed walk of length 0.
+       (Contributed by AV, 11-Feb-2022.) $)
+    0clwlkv $p |- ( ( X e. V /\ F = (/) /\ P : { 0 } --> { X } )
+                    -> F ( ClWalks ` G ) P ) $=
+      ( wcel c0 wceq cc0 csn wf w3a cclwlks cfv wbr cfz 3ad2ant1 wb cvv co fssd
```

Comment:
maybe `|- ( ( X e. V /\ P : { 0 } --> { X } ) -> (/)  ( ClWalks `` G ) P ) $= ` ?

Reply:
Since this theorem is for demonstration purposes only (not (yet) used in a proof), I want to keep this versio, because a walk, trail, path, etc. is represented by a pair <. F , P >. of (indexed) edges F and (indexed) vertices P.